Formula/tl-expected.rb

```diff
@@ -0,0 +1,54 @@
+class TlExpected < Formula
+  desc "C++11/14/17 std::expected with functional-style extensions"
+  homepage "https://github.com/TartanLlama/expected"
+  url "https://github.com/TartanLlama/expected/archive/refs/tags/v1.0.0.tar.gz"
+  sha256 "8f5124085a124113e75e3890b4e923e3a4de5b26a973b891b3deb40e19c03cee"
+  license "CC0-1.0"
+
+  depends_on "cmake" => :build
+
+  def install
+    args = %w[
+      -DEXPECTED_ENABLE_TESTS=OFF
+      -DCMAKE_INSTALL_LIBDIR=#{lib}
```

Comment:
This line will get overriden by `std_cmake_args`, so I suspect this does nothing. If you need it, use
```ruby
std_cmake_args(install_libdir: lib)
```
but as it is it doesn't seem like you need it.